src/ToricVarieties/LineBundles/auxilliary.jl

```diff
@@ -0,0 +1,36 @@
+
+function command_string(v::AbstractNormalToricVariety, c::Vector{fmpz})
+    # Initialize a list of strings, which we will eventually join to form the command string
+    string_list = Vector{String}()
+    
+    # Define helper function
+    joincomma(list) = join([string(x) for x in list], ",")
+    
+    # create dictionary from "our" variables to safe variable for use in cohomCalg
+    coordinate_names = [string(x) for x in Hecke.gens(cox_ring(v))]
+    coordinate_dictionary = Dict{String, String}(coordinate_names[i] => "x"*string(i) for i = 1:length(coordinate_names))
+    
+    # Add information about grading of Cox ring to string_list
+    divisors = gens(torusinvariant_divisor_group(v))
+    for i in 1:length(coordinate_names)
+        tmp = joincomma([x for x in map_from_weil_divisors_to_class_group(v)(divisors[i]).coeff])
```

Comment:
Why is the `[x for x in ...]` necessary? Can't you just write
```suggestion
        tmp = joincomma(map_from_weil_divisors_to_class_group(v)(divisors[i]).coeff)
```

Reply:
Thx. Done.